packages/@react-spectrum/menu/src/MenuItem.tsx

```diff
@@ -64,12 +84,18 @@ export function MenuItem<T>(props: MenuItemProps<T>) {
       onClose,
       closeOnSelect,
       isVirtualized,
-      onAction
+      onAction,
+      'aria-haspopup': isMenuDialogTrigger ? 'dialog' : undefined
     },
     state,
     ref
   );
-  let {hoverProps, isHovered} = useHover({isDisabled});
+  let endId = useSlotId();
+  let endProps: DOMAttributes = {};
+  if (endId) {
+    endProps.id = endId;
+    menuItemProps['aria-describedby'] = menuItemProps['aria-describedby'] + ' ' + endId;
```

Comment:
Looks like we are getting an undefined for the `menuItemProps['aria-describedby']` if the menu item doesn't have a description, doesn't come up as a issue from the announcement, but just something weird.
<img width="325" alt="image" src="https://github.com/adobe/react-spectrum/assets/9661127/4e141491-3fbb-42ef-ba58-d834f8faac83">